app/sentry.client.config.js

```diff
@@ -0,0 +1,19 @@
+// This file configures the initialization of Sentry on the browser.
+// The config you add here will be used whenever a page is visited.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs";
+
+const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN;
```

Comment:
do we need two separate ones?

Reply:
From my understanding only one.

Reply:
From my understanding only one environnement variable.

Another question is should we treat *app* and *site* as two different sentry projects, each one with its own dsn? I would say yes, since there is this notion of release tied to builds. Since app and site builds separately it would make more sense.